ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenterActivity.kt

```diff
@@ -0,0 +1,46 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.ui.Modifier
+import java.lang.ref.WeakReference
+
+class CustomerCenterActivity : ComponentActivity() {
```

Comment:
As discussed in another comment, I would make this internal.

Reply:
Also, we need to update the manifest with this activity to be able to use it I think?